delivery_multi_destination/tests/test_delivery_multi_destination.py

```diff
@@ -160,7 +160,8 @@ def test_delivery_multi_destination(self):
         sale_order_line = order.order_line.filtered("is_delivery")
         self.assertAlmostEqual(sale_order_line.price_unit, 50, 2)
         self.assertTrue(sale_order_line.is_delivery)
-        order.partner_shipping_id = self.partner_3.id
+        order.partner_shipping_id = self.partner_3
```

Comment:
Is modifying this line without the `.id` relevant for the test?

Reply:
No, it was a picky edit

Reply:
There are several of this FYI

Reply:
I still see them. Please reduce the diff undoing these changes.

Reply:
Yeah, fixed